root/etc/sudoers.d/50_nsapi_nethserver_blacklist

```diff
@@ -0,0 +1,17 @@
+#
+# nsapi_nethserver_blacklist
+#
+Cmnd_Alias NSAPI_NETHSERVER_BLACKLIST = \
+    /usr/libexec/nethserver/api/nethserver-blacklist/analysis/get_logs, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/analysis/read, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/analysis/validate, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/dashboard/execute, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/dashboard/read, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/dashboard/stats_dst_hosts, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/dashboard/stats_src_hosts, \
+    /usr/libexec/nethserver/api/nethserver-blacklist/dashboard/stats_total_hits, \
```

Comment:
Remove scripts which are not called directly from the UI.
All helpers scripts should not be listed here.